.kitchen.dokken.yml

```diff
@@ -68,4 +68,4 @@ platforms:
     pid_one_command: /bin/systemd
     intermediate_instructions:
       - RUN /usr/bin/apt-get update
-      - RUN /usr/bin/apt-get install apt-transport-https lsb-release procps net-tools -y
+      - RUN /usr/bin/apt-get install apt-transport-https lsb-release procps net-tools logrotate -y
```

Comment:
Why logrotate? Is there a link where someone fesses up to making this a requirement?

But if it's needed, it's needed.